infra/modules/worklytics-connector-specs/main.tf

```
@@ -258,9 +258,9 @@ locals {
       example_api_calls : [
         "/api/1.0/workspaces",
         "/api/1.0/users?workspace={ANY_WORKSPACE_GID}&limit=10",
-        "/api/1.0/workspaces/{ANY_WORKSPACE_GID}/teams&limit=10",
+        "/api/1.0/workspaces/{ANY_WORKSPACE_GID}/teams?limit=10",
         "/api/1.0/teams/{ANY_TEAM_GID}/projects?limit=20",
-        "/api/1.0/tasks?project={ANY_PROJECT_GID}",
+        "/api/1.0/tasks?project={ANY_PROJECT_GID}&limit=10",
         "/api/1.0/tasks/{ANY_TASK_GID}",
         "/api/1.0/tasks/{ANY_TASK_GID}/stories",
```

Following your change:

```suggestion
        "/api/1.0/tasks/{ANY_TASK_GID}/stories?limit=10",
```